docs/fundamentals/code-analysis/quality-rules/snippets/csharp/all-rules/.editorconfig

```diff
@@ -0,0 +1,18 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+root = true
+
+# All files
+[*]
+indent_style = space
+
+# CS0659: Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
+dotnet_diagnostic.CS0659.severity = suggestion
+
+# CAXXXX analyzers
+dotnet_diagnostic.CA1822.severity = none
+
+# CS0108: Member hides inherited member; missing new keyword
+dotnet_diagnostic.CS0108.severity = suggestion
+
+[*.xml]
+indent_size = 2
```

Comment:
There is already an .editorconfig in repo root. I'm not sure why a new one is needed?